### PR TITLE
Schema to_hash method

### PR DESCRIPTION
**Pull request by Ladas**
Schema to_hash method

### CODE REVIEW

File: lib/topological_inventory/schema/default.rb [hunk 1]
```diff
@@ -37,6 +37,15 @@ def initialize_inventory_collections
         add_cross_link_vms
       end
 
+      def to_hash
+        {
+          "schema": {
+            "name": "Default"
```
Comment (agrare) [inline lib/topological_inventory/schema/default.rb]: Can we get the schema name dynamically that way if we have other schemas that inherit from this one they can just the same method?

Comment (Ladas) [reply lib/topological_inventory/schema/default.rb]: I think we can do that once we have some ind of base schema?

Comment (Ladas) [reply lib/topological_inventory/schema/default.rb]: or should I just put that to persister?

Comment (agrare) [reply lib/topological_inventory/schema/default.rb]: No I think here makes sense, so can you add a `TopologicalInventory::Schema::Base` with this there?  Not sure if any of the other helper methods make sense in base but some might.

**Pull request merged**